WordPress/Tests/XSS/EscapeOutputUnitTest.php

```diff
@@ -63,6 +63,7 @@ public function getErrorList() {
 			205 => 1,
 			206 => 1,
 			207 => 1,
+			212 => ( PHP_VERSION_ID < 50300 ) ? 1 : 0, // PHPCS on PHP 5.2 does not recognize T_NOWDOC.
```

Comment:
I see this version compare being used quite a bit. Would it make sense to have a constant defined for this magic number? Or have a `is_php_at_least_53()` function which returns a boolean?